docs/source/caching.mdx

```diff
@@ -177,3 +177,38 @@ store.withinReadWriteTransaction({ transaction in
   }
 })
 ```
+
+### delete
+
+Delete functionality is limited at this time. There are presently two options available on `NormalizedCache` which are supported by the included implementations of it: 
```

Comment:
> There are presently two options available on `NormalizedCache` which are supported by the included implementations of it
This sentence seems a bit hard to understand, and I would avoid `NormalizedCache`. Maybe something like:
> There are presently two deletion methods available on `ApolloStore`, which are supported by both the in memory and the SQLite cache.